CMakeLists.txt

```diff
@@ -294,8 +299,10 @@ ENDIF()
 
 # Sources.
 
+FILE(GLOB binaryen_HEADERS src/*.h)
 set(binaryen_SOURCES
   src/binaryen-c.cpp
+  ${binaryen_HEADERS}
```

Comment:
Is this mostly for IDE integration or is there some other effect?  e..g does it effect `install`?

Reply:
Yes, just for IDEs. Moved to other PR.